components/application-broker/internal/broker/provision_test.go

```diff
@@ -190,7 +171,6 @@ func TestProvisionAsync(t *testing.T) {
 func TestProvisionProcessWithKnative(t *testing.T) {
```

Comment:
Do we still need this test `TestProvisionProcessWithKnative`?

Reply:
its gone